app/src/main/java/org/oppia/app/topic/review/ReviewSkillSelector.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.topic.review
+
+import org.oppia.app.model.SkillSummary
+
+/** Listener for topic review-skill click. */
```

Comment:
Suggest: ``Listener for when a skill is selected for review.``

Reply:
Updated to: `/** Listener for when a skill is selected for review. */`